music/src/core/player.ts

```diff
@@ -119,10 +134,12 @@ export abstract class BasePlayer {
    * @returns a Promise that resolves when playback is complete.
    */
   start(seq: INoteSequence, qpm?: number): Promise<void> {
-    if (this.playClick) {
+    let isQuantized = sequences.isQuantizedSequence(seq);
+    if (this.playClick && isQuantized) {
       seq = this.makeClickSequence(seq);
     }
-    if (sequences.isQuantizedSequence(seq)) {
+    Tone.Transport.bpm.value = 120;
```

Comment:
Why 120 here? Shouldn't it be set to the QPM of the unquantized sequence?

Reply:
what i thought i had to do for this to work was to first select a "standard" qpm (i picked 120 because that's what's in constants.DEFAULT_QUARTERS_PER_MINUTE), enqueue the notes, and then switch to the desired qpm.

i guess i was also assuming an unquantized sequence shouldn't include qpm (the one in my example doesn't). do unquantized sequences also normally specify qpm in tempos[0].qpm?

Reply:
You should use the tempos[0].qpm if it exists, or the default if it does not.

Reply:
done.